src/stores/appcatalog/actions.ts

```diff
@@ -313,8 +316,15 @@ function fixTestAppReadmeURLs(readmeURL: string): string {
    * doesn't match, then the string is returned as is.
    * https://regex101.com/r/K2dxdN/1
    */
-  const regexMatcher = /^(.*)\/v?[0-9]+\.[0-9]+\.[0-9]+-(.*)\/README\.md$/;
-  const fixedReadmeURL = readmeURL.replace(regexMatcher, '$1/$2/README.md');
+
+  const escapedReadmeFile = Constants.README_FILE.replace('.', '.');
```

Comment:
How does this change the value? 😬

Reply:
Lol I think I was testing to make sure the tests would fail, but didn't put it back to the right value.

in this case the test still pass because of the meaning of '.' in regex.

Fixed :) thanks for spotting it

Reply:
Ohh no.. it's prettier that's doing it... hmm

Reply:
Ah ok I wasn't escaping the escape character correctly.

Reply:
What I meant was that I'm not sure why we are replacing dots with dots. Am I missing something?